dandi/files/bases.py

```diff
@@ -12,12 +12,15 @@
 from threading import Lock
 from typing import Any, BinaryIO, Generic, Optional
 from xml.etree.ElementTree import fromstring
+import json
+from packaging import version
```

Comment:
I can tell that you didn't run pre-commit on this code.  Please do `pip install pre-commit`, then run `pre-commit install` in your clone of dandi, and then `pre-commit run -a` to ensure all your contributions so far have been formatted, linted, & isorted.

Reply:
Done

Have y'all thought about using https://pre-commit.ci/ to automatically resolve such things? Since we've started using it it's _so nice_ to never having to think about